practice-app/app/views.py

```diff
@@ -242,3 +252,48 @@ def worldTime():
                 languageCode=world_time.languageCode,
             )
         return render_template("worldtime.html")
+
+@app.route("/pokemon", methods=["GET", "POST"])
+def PokemonPage():
```

Comment:
Move to another file.

Use snake case, since this is a function - not a class name

Reply:
Moved to "pokemon_api.py". thanks for your suggestion.